R/writedata.R

```diff
@@ -1,5 +1,92 @@
 #### Output functions
 
+#' Subset a trained emulator.
+#'
+#' A trained fldgen emulator features a large amount of data for both
+#' using the emulator and rigorously validating an emulator.
+#'
+#' If one is just interested in the use of an emulator for generating
+#' felds, this function can be called to reduce a trained emulator to
+#' the bare essential list entries, which can then be saved and called
+#' the same as an unreduced emulator by generate.TP.resids and
+#' generate.TP.fullgrids
+#'
+#' Note that with this reduced emulator, there is NO way to reconstruct
+#' the training data. A fully trained emulator contains a copy of the
+#' training data, in addition to the training regressor values (tgav),
+#' and the estimated linear model parameters and residuals
+#'  (meanfieldT$b, w, r), which together can also reconstruct the data.
+#'
+#' Even though the coordinate information stored in an emulator$griddataT
+#' is not needed directly to generate a new field of residuals or full data,
+#' it is often needed in downstream use of the fields. Therfore an entry
+#' reducedEmulator$griddataT$coord containg a matrix is saved in the
+#' reducedEmulator. Each is a matrix of coordinates for each grid cell, with
+#' cells in rows and latitude, longitude in the two columns. Keeping these
+#' coordinate matrices for T and P is negligible.
+#'
+#' Finally, the reduced emulator produced by this function is specifically
+#' meant for temperature and precipitation only, and is not robust to
+#' extension to other variables.
+#'
+#' Finally finally, if a user is interested in a different subset of
+#' list entries in a trained emulator, they are encouraged to subset and
+#' save themself, as appropriate for their project.
+#'
+#' @param emulator A trained fldgen emulator, with all entries needed
+#' for generating new residuals and for rigourously validating the
+#' quality of the trained emulator
+#'
+#' @return reducedEmulator A trained fldgen emulator with only the list
+#' entries needed by generate.TP.resids and generate.TP.fullgrids for
+#' generating new fields
```

Comment:
Bleh I hate to suggest it but would it be helpful to include documentation about is included in the reduced emulator object? That might be more helpful to users than simply saying `entries needed by generate.TP.resids and generate.TP.fullgrids` yes that is an important point but at the same time idk what those things are and would have to go digging to figure that out... 

```
@return A list of structures.
#' \describe{
#' \item{runstats}{Run statistics such as acceptance rate and effective number of samples.
#' These apply to the merged set of runs for the configuration.}
#' \item{mcobjs}{List of lists of \code{metrosamp} objects.  Each list in the top level
#' corresponds to a configuration; the elements in the second-level list are the objects
#' for the individual runs in the configuration.}
#' \item{codaobjs}{\code{\link[coda]{mcmc.list}} object for each configuration.  These objects
#' represent the merged results of all of the runs in a configuration in a form that can be used
#' with the utilities in the coda package.}
#' }
```